nrf_rpc/nrf_rpc.c

```diff
@@ -237,6 +264,45 @@ static int simple_send(const struct nrf_rpc_group *group, uint8_t dst, uint8_t t
 	return send(group->transport, tx_buf, NRF_RPC_HEADER_SIZE + len);
 }
 
+static int group_init_send(const struct nrf_rpc_group *group)
+{
+	struct protocol_version *protocol_version;
+	struct header hdr;
+	uint8_t *tx_buf;
+	uint8_t *packet;
+	size_t len;
+
+	hdr.dst = NRF_RPC_ID_UNKNOWN;
+	hdr.type = NRF_RPC_PACKET_TYPE_INIT;
+	hdr.id = 0;
+	hdr.src_group_id = group->data->src_group_id;
+	hdr.dst_group_id = NRF_RPC_ID_UNKNOWN;
+
+	len = strlen(group->strid) + NRF_RPC_PROTOCOL_VERSION_FIELD_SIZE;
+
+	nrf_rpc_alloc_tx_buf(group, &tx_buf, len);
+	if (tx_buf == NULL) {
+		return -NRF_ENOMEM;
+	}
+
+	packet = tx_buf;
+	tx_buf -= NRF_RPC_HEADER_SIZE;
```

Comment:
Please add a comment explaining that buffer allocation includes header space.

Reply:
Sure. Added